config.toml

```diff
@@ -4,7 +4,7 @@
 baseURL = "https://seismo-learn.org/links/"
 languageCode = "en-us"
 title = "Seismo Links"
-copyright = "&copy; Copyright 2018 - {year}, seismo-learn."
+copyright = "&copy; Copyright 2018 - {year}, [seismo-learn](https://seismo-learn.org/)."
```

Comment:
Add a link to https://github.com/seismo-learn/links#contributing for contributing guide?

Reply:
Done in 2642d6a6be7ce67ed4bfb6050aa8bc3b1abe141e